src/forms/components/Form.js

```diff
@@ -113,43 +114,50 @@ const Form = props => {
         onSubmit={handleSubmit(onSubmit)}
         sx={{ width: '100%' }}
       >
-        {fields.map(field =>
-          field.notControlledRender ? (
-            <React.Fragment key={field.name}>
-              {field.notControlledRender()}
-            </React.Fragment>
-          ) : (
-            <Grid
-              key={field.name}
-              item
-              xs={12}
-              {..._.get('props.gridItemProps', field)}
-              sx={{ paddingBottom: 3 }}
-            >
-              <Controller
-                name={field.name}
-                control={control}
-                render={controllerProps => (
-                  <FormField
-                    field={controllerProps.field}
-                    fieldState={controllerProps.fieldState}
-                    required={_.includes(field.name, requiredFields)}
-                    id={`${prefix}-${field.name}`}
-                    label={field.label}
-                    info={field.info}
-                    inputProps={{
-                      type: field.type || 'text',
-                      placeholder: t(field.placeholder),
-                      ..._.getOr({}, 'props.inputProps', field),
-                    }}
-                    localizationPrefix={localizationPrefix}
-                    {..._.getOr({}, 'props', field)}
-                  />
-                )}
-              />
-            </Grid>
-          )
-        )}
+        {fields
+          .filter(field => {
+            if (!filterField) {
+              return true;
+            }
+            return filterField(field, formProps);
```

Comment:
would 
```php
return filterField ? filterField(field, formProps) : true;
```
or something similar make sense? Or is that too confusing?

Reply:
This makes sense too me, and personally is more readable, I will change it